cmd/compute-domain-kubelet-plugin/device_state.go

```diff
@@ -372,6 +368,11 @@ func (s *DeviceState) applyComputeDomainDaemonConfig(ctx context.Context, config
 		return nil, fmt.Errorf("only expected 1 device for requests '%v' in claim '%v'", requests, claim.UID)
 	}
 
+	// Add info about this node to the ComputeDomain status.
```

Comment:
nit: The comment and the method name don't seem to quite align. I would expect something like `AddNodeStatusToComputeDomain` to better indicate intent.

Reply:
I don't feel strongly -- made the change